README.md

```diff
@@ -85,179 +81,17 @@ A hardware TPM should always be used when real secrets and trust is required.
-configured as the default in Keylime. On older kernel versions it is 
-recommended to use the [tpm2-abrmd](https://github.com/tpm2-software/tpm2-abrmd) 
-resource manager.
-
-Alternatively, it is also possible, though not recommended, to communicate
-directly with the TPM (and not use a resource manager).  This can be done by
-setting the environment var `TPM2TOOLS_TCTI` to the appropriate value:
-
-To talk directly to the swtpm2 emulator: 
-
-`export TPM2TOOLS_TCTI="mssim:port=2321"`
-
-To talk directly to a real TPM: 
-
-`export TPM2TOOLS_TCTI="device:/dev/tpm0"`
-
-#### Install Keylime
-
-You're finally ready to install keylime!
-
-```bash
-sudo python3 -m pip install . -r requirements.txt
-```
-
-## Making sure your TPM is ready for keylime
-
-The above instructions for installing the TPM libraries will be configured
-to talk to `/dev/tpm0`.  If this device is not on your system, then you may need
-to build/install TPM support for your kernel.  You can use following command
-to see if the kernel is initializing the TPM driver during boot:
-
-`dmesg | grep -i tpm`
-
-If you have the `/dev/tpm0` device, you next need to get it into the right state. The kernel
-driver reports status on the TPM in `/sys`.  You can locate the folder with relevant
-info from the driver using:
-
-`sudo find /sys -name tpm0`
-
-Several results may be returned, but the duplicates are just symlinks to one
-location.  Go to one of the returned paths, for example, `/sys/class/misc/tpm0`.  Now
-change to the device directory.  Here you can find some information from the TPM like
-the current pcr values and sometimes the public EK is available.  It will also report
-two important state values: active and enabled.  To use keylime, both of these must
-be 1.  If they are not, you may need to reboot into the BIOS to enable and activate
-the TPM.  If you need to both enable and activate, then you must enable first, reboot,
-then activate and finally reboot again.  It is also possible that you may need to
-assert physical presence (see manual for your system on how to do this) in order to
-accomplish these actions in your BIOS.
-
-If your system shows enabled and active, you can next check the "owned" status
-in the /sys directory. The [sysfs ABI](https://www.kernel.org/doc/Documentation/ABI/stable/sysfs-class-tpm) lists
-where the kernel populates these entries. Keylime can take a system that is not
-owned (i.e., owned = 0) and take control of it.  Keylime can also take a system
-that is already owned, provided that you know the owner password and that
-keylime or another trusted computing system that relies upon tpm4720 previously
-took ownership.  If you know the owner password, you can set the option
-`tpm_ownerpassword` in `keylime.conf` to this known value.
+Multiple ways of installing Keylime can be found in the documentation: [Installation of Keylime](https://keylime.readthedocs.io/en/latest/installation.html).
```

Comment:
I would refrase it a bit different: "To install Keylime refers to the [documentation](https://keylime.readthedocs.io/en/latest/installation.html)."

Reply:
yeah changed it